allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/interceptors/testrun/TestRunStateHolder.kt

```diff
@@ -0,0 +1,32 @@
+package com.kaspersky.components.alluresupport.interceptors.testrun
+
+import java.io.File
+
+/**
+ * Used to store shared state used by multiple interceptors
+ */
+class TestRunStateHolder {
```

Comment:
Please use more correct name, this name to mush general, but this holder stores only video file references